src/lang/en-US/LoggingIn.ts

```diff
@@ -110,6 +110,11 @@ export const LoggingIn: Record<string, string> = {
 
     'tenant.error.failedToFetch.message': `There was an issue while checking if you have access to a tenant.`,
 
+    'tenant.usedSso.title': `Successfully Authenticated`,
+    'tenant.usedSso.message': `Your account was created successfully and can be added to your organization's ${CommonMessages.company} tenant.`,
+    'tenant.usedSso.instructions': `To do so, ask an ${CommonMessages.company} admin at your company to go to the URL below and create an invite link which they can share with you.`,
+    'tenant.usedSso.instructions.fullPath': `https://dashboard.estuary.dev/admin/accessGrants`,
```

Comment:
Should a comment announcing the existence of this URL content be added to the file in which this route lives in an effort to keep the two in sync?